packages/pnpm/src/cmd/outdated.ts

```diff
@@ -47,35 +47,45 @@ for (let [key, value] of Object.entries(TABLE_OPTIONS.border)) {
   TABLE_OPTIONS.border[key] = chalk.grey(value)
 }
 
+/**
+ * Default comparators used as the argument to `ramda.sortWith()`.
+ */
+export const DEFAULT_COMPARATORS = [
+  sortBySemverChange,
+  (o1: OutdatedWithVersionDiff, o2: OutdatedWithVersionDiff) => o1.packageName.localeCompare(o2.packageName),
+] as const
+
+export interface OutdatedOptions {
```

Comment:
instead of declaring the types for every field, you can create this type by using Pick<PnpmConfigs, 'alwaysAuth' | 'ca' | ...>
PnpmConfig is exported from `@pnpm/config`